pkg/xds/auth/universal/authenticator.go

```diff
@@ -82,6 +84,31 @@ func (u *universalAuthenticator) authZoneIngress(ctx context.Context, zoneIngres
 	return nil
 }
 
+func (u *universalAuthenticator) authZoneEgress(
+	ctx context.Context,
+	_ *core_mesh.ZoneEgressResource,
```

Comment:
it's a private function. If we don't need ZoneEgressResource, let's remove it from the signature

Reply:
addressed